tests/test_file.py

```diff
@@ -45,6 +45,18 @@ def test_locale(self, mozwebqa):
             print "There is no language selector on the page"
 
     @pytest.mark.nondestructive
+    def test_locale_zh_TW (self, mozwebqa):
+        main_page = MySiteHomePage(mozwebqa)
+        zh_TW_locale = ['/zh-TW']
+        for path in zh_TW_locale:
+            url = main_page.base_url + path
+
+            Assert.true(url.endswith('/en-US'))
```

Comment:
This will fail because it's testing the above variable. What we need to do is make a request based on the new URL and check that our final URL after any redirects contains either our original locale or 'en-US'. We could make this request via Selenium or something like requests.